### PR TITLE
Fixes #17917 - Use digest to detect tailoring_file changes

### PR DESCRIPTION
**Pull request by xprazak2**
TODO:
- [x] rebase onto #39

### CODE REVIEW

**Comment by ares** [comment]
This probably needs to be rebased on top of #41 and should be merged right after it.

**Comment by ares** [comment]
please also update the commit message, this is now only for tailoring files

**Comment by xprazak2** [comment]
I updated the commit message

**Comment by ares** [comment]
@xprazak2 needs rebasing now

File: test/fetch_scap_api_test.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def setup
     Proxy::OpenSCAP::Plugin.settings.stubs(:spooldir).returns(@results_path)
     Proxy::OpenSCAP::Plugin.settings.stubs(:reportsdir).returns(@results_path)
     @scap_content = File.new("#{Dir.getwd}/test/data/ssg-rhel7-ds.xml").read
+    @digest = Digest::SHA256.hexdigest @scap_content
```
Comment (ares) [inline test/fetch_scap_api_test.rb]: probably not needed, the same line is 2 lines below

**Comment by ares** [comment]
Thanks @xprazak2, merged.

**Pull request merged**